lib/ops/opsservice/report.go

```diff
@@ -145,17 +145,18 @@ func (s *site) getReport(runner remoteRunner, servers []remoteServer, masters []
 		}
 		serverRunner := &serverRunner{server: server, runner: runner}
 		reportWriter := getReportWriterForServer(dir, server)
+		logger := log.WithField("server", server.Address())
 		if err := s.collectKubernetesInfo(reportWriter, serverRunner, since); err != nil {
-			log.WithError(err).Error("Failed to collect Kubernetes info.")
+			logger.WithError(err).Error("Failed to collect Kubernetes info.")
 		}
 		if err := s.collectEtcdInfoFromMasters(dir, masters, runner); err != nil {
-			log.WithError(err).Error("Failed to collect etcd info.")
+			logger.WithError(err).Error("Failed to collect etcd info.")
```

Comment:
Use `log` instead of `logger` since we are collecting etcd info from all masters instead of the just first selected server.

Reply:
Will revert this change.